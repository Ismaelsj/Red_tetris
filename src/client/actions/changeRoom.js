import { ROOM_CHANGE } from '../event/reduxEvent'

export const changeRoom = (newRoom, roomPlayers, pieces) => {
    return {
        type: ROOM_CHANGE,
        roomName: newRoom,
        players: roomPlayers,
        pieces: pieces
    }
}