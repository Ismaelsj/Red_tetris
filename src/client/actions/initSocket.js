import { INIT_SOCKET } from '../event/reduxEvent'

export const initSocket = (socket) => {
    return {
        type: INIT_SOCKET,
        socket: socket,
    }
}