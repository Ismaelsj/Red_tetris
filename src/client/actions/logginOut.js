import { LOGOUT_LOGGINOUT } from '../event/reduxEvent'

export const logginOutUser = () => {
    return {
        type: LOGOUT_LOGGINOUT,
    }
}