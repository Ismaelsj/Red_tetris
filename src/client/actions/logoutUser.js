import { LOGOUT_LOGOUT } from '../event/reduxEvent'

export const logoutUser = () => {
    return {
        type: LOGOUT_LOGOUT,
    }
}