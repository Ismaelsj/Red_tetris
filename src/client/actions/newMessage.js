import { MESSAGE_NEW_MESSAGE } from '../event/reduxEvent'

export const newMessage = (message) => {
    return {
        type: MESSAGE_NEW_MESSAGE,
        message: message
    }
}