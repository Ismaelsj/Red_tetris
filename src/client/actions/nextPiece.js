import { GAME_MEXT_PIECE } from '../event/reduxEvent'

export const nextPiece = () => {
    return {
        type: GAME_MEXT_PIECE
    }
}