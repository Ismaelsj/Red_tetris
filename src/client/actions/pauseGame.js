import { GAME_PAUSE } from '../event/reduxEvent'

export const pauseGame = () => {
    return {
        type: GAME_PAUSE
    }
}