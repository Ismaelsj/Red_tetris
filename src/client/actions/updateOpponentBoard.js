import { GAME_UPDATE_OPPONENT_BOARD} from '../event/reduxEvent'

export const updateOpponentBoard = (newBoard) => {
    return {
        type: GAME_UPDATE_OPPONENT_BOARD,
        newBoard: newBoard
    }
}