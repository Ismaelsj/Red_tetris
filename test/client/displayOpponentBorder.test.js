import 'jsdom-global/register';
import React from 'react'
import { mount } from 'enzyme';
import "babel-polyfill"

import DisplayOpponentBorder from '../../src/client/components/displayBoard/displayOpponentBorder'

import { Provider } from 'react-redux'
import io from 'socket.io-client';
import storeConfig from '../../src/client/middlewares/store'

import { updateOpponentBoard } from '../../src/client/actions/updateOpponentBoard'

const serverParams = {
    host: '0.0.0.0',
    port: 8882,
    url: 'http://' + '0.0.0.0' + ':' + '8882' 
}

const { url } = serverParams

const socket = io(url)

const templateEmptyBoard = [
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
];

const initialStore = {
    error: {
        intenalError: false
    },
    user: {
        userName: 'Ismael',
        isAuth: false,
        socket: null,
        room: 'Ismael',
    },
    room: {
        roomName: 'Ismael',
        userName: 'Ismael',
        players: ['Ismael'],
        isManager: false,
        conversation: [],
        isChanging: false,
        msgNot: 0,
    },
    game: {
        Game: null,
        Score: 0,
        Malus: 0,
        isPlaying: false,
        Pieces: [],
        GameHistory: [],
        GameBoard: {
            player: templateEmptyBoard,
            opponent: templateEmptyBoard
        }
    },
    login: {
        socket: null,
        logginOut: false,
        userName: 'Ismael',
        roomName: 'Ismael',
        isValid: true,
        isUsed: false,
        errMsg: 'Scucess'
    },
    changeRoom: {
        isValid: false,
        errMsg: ''
    }
}

const store = storeConfig(socket, initialStore)

store.dispatch(updateOpponentBoard(templateEmptyBoard))

describe('Chat', () => {
    it('chat => redner: props -> ...', () => {
        const component = mount(
            <Provider store={store}>
                <DisplayOpponentBorder />,
            </Provider>
        );
        expect(component).toMatchSnapshot();
    });
});