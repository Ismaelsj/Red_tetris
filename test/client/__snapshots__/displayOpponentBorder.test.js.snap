// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat chat => redner: props -> ... 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(DisplayOpponentBorder)>
    <DisplayOpponentBorder
      dispatch={[Function]}
      game={
        Object {
          "Game": null,
          "GameBoard": Object {
            "opponent": Array [
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
            ],
            "player": Array [
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
              Array [
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
                0,
              ],
            ],
          },
          "GameHistory": Array [],
          "Malus": 0,
          "Pieces": Array [],
          "Score": 0,
          "isPlaying": false,
        }
      }
    >
      <div>
        <div
          className="rows"
          id={0}
          key="0"
        >
          <div
            className="columns"
            id="0-0"
            key="0"
          />
          <div
            className="columns"
            id="0-1"
            key="1"
          />
          <div
            className="columns"
            id="0-2"
            key="2"
          />
          <div
            className="columns"
            id="0-3"
            key="3"
          />
          <div
            className="columns"
            id="0-4"
            key="4"
          />
          <div
            className="columns"
            id="0-5"
            key="5"
          />
          <div
            className="columns"
            id="0-6"
            key="6"
          />
          <div
            className="columns"
            id="0-7"
            key="7"
          />
          <div
            className="columns"
            id="0-8"
            key="8"
          />
          <div
            className="columns"
            id="0-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={1}
          key="1"
        >
          <div
            className="columns"
            id="1-0"
            key="0"
          />
          <div
            className="columns"
            id="1-1"
            key="1"
          />
          <div
            className="columns"
            id="1-2"
            key="2"
          />
          <div
            className="columns"
            id="1-3"
            key="3"
          />
          <div
            className="columns"
            id="1-4"
            key="4"
          />
          <div
            className="columns"
            id="1-5"
            key="5"
          />
          <div
            className="columns"
            id="1-6"
            key="6"
          />
          <div
            className="columns"
            id="1-7"
            key="7"
          />
          <div
            className="columns"
            id="1-8"
            key="8"
          />
          <div
            className="columns"
            id="1-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={2}
          key="2"
        >
          <div
            className="columns"
            id="2-0"
            key="0"
          />
          <div
            className="columns"
            id="2-1"
            key="1"
          />
          <div
            className="columns"
            id="2-2"
            key="2"
          />
          <div
            className="columns"
            id="2-3"
            key="3"
          />
          <div
            className="columns"
            id="2-4"
            key="4"
          />
          <div
            className="columns"
            id="2-5"
            key="5"
          />
          <div
            className="columns"
            id="2-6"
            key="6"
          />
          <div
            className="columns"
            id="2-7"
            key="7"
          />
          <div
            className="columns"
            id="2-8"
            key="8"
          />
          <div
            className="columns"
            id="2-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={3}
          key="3"
        >
          <div
            className="columns"
            id="3-0"
            key="0"
          />
          <div
            className="columns"
            id="3-1"
            key="1"
          />
          <div
            className="columns"
            id="3-2"
            key="2"
          />
          <div
            className="columns"
            id="3-3"
            key="3"
          />
          <div
            className="columns"
            id="3-4"
            key="4"
          />
          <div
            className="columns"
            id="3-5"
            key="5"
          />
          <div
            className="columns"
            id="3-6"
            key="6"
          />
          <div
            className="columns"
            id="3-7"
            key="7"
          />
          <div
            className="columns"
            id="3-8"
            key="8"
          />
          <div
            className="columns"
            id="3-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={4}
          key="4"
        >
          <div
            className="columns"
            id="4-0"
            key="0"
          />
          <div
            className="columns"
            id="4-1"
            key="1"
          />
          <div
            className="columns"
            id="4-2"
            key="2"
          />
          <div
            className="columns"
            id="4-3"
            key="3"
          />
          <div
            className="columns"
            id="4-4"
            key="4"
          />
          <div
            className="columns"
            id="4-5"
            key="5"
          />
          <div
            className="columns"
            id="4-6"
            key="6"
          />
          <div
            className="columns"
            id="4-7"
            key="7"
          />
          <div
            className="columns"
            id="4-8"
            key="8"
          />
          <div
            className="columns"
            id="4-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={5}
          key="5"
        >
          <div
            className="columns"
            id="5-0"
            key="0"
          />
          <div
            className="columns"
            id="5-1"
            key="1"
          />
          <div
            className="columns"
            id="5-2"
            key="2"
          />
          <div
            className="columns"
            id="5-3"
            key="3"
          />
          <div
            className="columns"
            id="5-4"
            key="4"
          />
          <div
            className="columns"
            id="5-5"
            key="5"
          />
          <div
            className="columns"
            id="5-6"
            key="6"
          />
          <div
            className="columns"
            id="5-7"
            key="7"
          />
          <div
            className="columns"
            id="5-8"
            key="8"
          />
          <div
            className="columns"
            id="5-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={6}
          key="6"
        >
          <div
            className="columns"
            id="6-0"
            key="0"
          />
          <div
            className="columns"
            id="6-1"
            key="1"
          />
          <div
            className="columns"
            id="6-2"
            key="2"
          />
          <div
            className="columns"
            id="6-3"
            key="3"
          />
          <div
            className="columns"
            id="6-4"
            key="4"
          />
          <div
            className="columns"
            id="6-5"
            key="5"
          />
          <div
            className="columns"
            id="6-6"
            key="6"
          />
          <div
            className="columns"
            id="6-7"
            key="7"
          />
          <div
            className="columns"
            id="6-8"
            key="8"
          />
          <div
            className="columns"
            id="6-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={7}
          key="7"
        >
          <div
            className="columns"
            id="7-0"
            key="0"
          />
          <div
            className="columns"
            id="7-1"
            key="1"
          />
          <div
            className="columns"
            id="7-2"
            key="2"
          />
          <div
            className="columns"
            id="7-3"
            key="3"
          />
          <div
            className="columns"
            id="7-4"
            key="4"
          />
          <div
            className="columns"
            id="7-5"
            key="5"
          />
          <div
            className="columns"
            id="7-6"
            key="6"
          />
          <div
            className="columns"
            id="7-7"
            key="7"
          />
          <div
            className="columns"
            id="7-8"
            key="8"
          />
          <div
            className="columns"
            id="7-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={8}
          key="8"
        >
          <div
            className="columns"
            id="8-0"
            key="0"
          />
          <div
            className="columns"
            id="8-1"
            key="1"
          />
          <div
            className="columns"
            id="8-2"
            key="2"
          />
          <div
            className="columns"
            id="8-3"
            key="3"
          />
          <div
            className="columns"
            id="8-4"
            key="4"
          />
          <div
            className="columns"
            id="8-5"
            key="5"
          />
          <div
            className="columns"
            id="8-6"
            key="6"
          />
          <div
            className="columns"
            id="8-7"
            key="7"
          />
          <div
            className="columns"
            id="8-8"
            key="8"
          />
          <div
            className="columns"
            id="8-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={9}
          key="9"
        >
          <div
            className="columns"
            id="9-0"
            key="0"
          />
          <div
            className="columns"
            id="9-1"
            key="1"
          />
          <div
            className="columns"
            id="9-2"
            key="2"
          />
          <div
            className="columns"
            id="9-3"
            key="3"
          />
          <div
            className="columns"
            id="9-4"
            key="4"
          />
          <div
            className="columns"
            id="9-5"
            key="5"
          />
          <div
            className="columns"
            id="9-6"
            key="6"
          />
          <div
            className="columns"
            id="9-7"
            key="7"
          />
          <div
            className="columns"
            id="9-8"
            key="8"
          />
          <div
            className="columns"
            id="9-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={10}
          key="10"
        >
          <div
            className="columns"
            id="10-0"
            key="0"
          />
          <div
            className="columns"
            id="10-1"
            key="1"
          />
          <div
            className="columns"
            id="10-2"
            key="2"
          />
          <div
            className="columns"
            id="10-3"
            key="3"
          />
          <div
            className="columns"
            id="10-4"
            key="4"
          />
          <div
            className="columns"
            id="10-5"
            key="5"
          />
          <div
            className="columns"
            id="10-6"
            key="6"
          />
          <div
            className="columns"
            id="10-7"
            key="7"
          />
          <div
            className="columns"
            id="10-8"
            key="8"
          />
          <div
            className="columns"
            id="10-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={11}
          key="11"
        >
          <div
            className="columns"
            id="11-0"
            key="0"
          />
          <div
            className="columns"
            id="11-1"
            key="1"
          />
          <div
            className="columns"
            id="11-2"
            key="2"
          />
          <div
            className="columns"
            id="11-3"
            key="3"
          />
          <div
            className="columns"
            id="11-4"
            key="4"
          />
          <div
            className="columns"
            id="11-5"
            key="5"
          />
          <div
            className="columns"
            id="11-6"
            key="6"
          />
          <div
            className="columns"
            id="11-7"
            key="7"
          />
          <div
            className="columns"
            id="11-8"
            key="8"
          />
          <div
            className="columns"
            id="11-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={12}
          key="12"
        >
          <div
            className="columns"
            id="12-0"
            key="0"
          />
          <div
            className="columns"
            id="12-1"
            key="1"
          />
          <div
            className="columns"
            id="12-2"
            key="2"
          />
          <div
            className="columns"
            id="12-3"
            key="3"
          />
          <div
            className="columns"
            id="12-4"
            key="4"
          />
          <div
            className="columns"
            id="12-5"
            key="5"
          />
          <div
            className="columns"
            id="12-6"
            key="6"
          />
          <div
            className="columns"
            id="12-7"
            key="7"
          />
          <div
            className="columns"
            id="12-8"
            key="8"
          />
          <div
            className="columns"
            id="12-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={13}
          key="13"
        >
          <div
            className="columns"
            id="13-0"
            key="0"
          />
          <div
            className="columns"
            id="13-1"
            key="1"
          />
          <div
            className="columns"
            id="13-2"
            key="2"
          />
          <div
            className="columns"
            id="13-3"
            key="3"
          />
          <div
            className="columns"
            id="13-4"
            key="4"
          />
          <div
            className="columns"
            id="13-5"
            key="5"
          />
          <div
            className="columns"
            id="13-6"
            key="6"
          />
          <div
            className="columns"
            id="13-7"
            key="7"
          />
          <div
            className="columns"
            id="13-8"
            key="8"
          />
          <div
            className="columns"
            id="13-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={14}
          key="14"
        >
          <div
            className="columns"
            id="14-0"
            key="0"
          />
          <div
            className="columns"
            id="14-1"
            key="1"
          />
          <div
            className="columns"
            id="14-2"
            key="2"
          />
          <div
            className="columns"
            id="14-3"
            key="3"
          />
          <div
            className="columns"
            id="14-4"
            key="4"
          />
          <div
            className="columns"
            id="14-5"
            key="5"
          />
          <div
            className="columns"
            id="14-6"
            key="6"
          />
          <div
            className="columns"
            id="14-7"
            key="7"
          />
          <div
            className="columns"
            id="14-8"
            key="8"
          />
          <div
            className="columns"
            id="14-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={15}
          key="15"
        >
          <div
            className="columns"
            id="15-0"
            key="0"
          />
          <div
            className="columns"
            id="15-1"
            key="1"
          />
          <div
            className="columns"
            id="15-2"
            key="2"
          />
          <div
            className="columns"
            id="15-3"
            key="3"
          />
          <div
            className="columns"
            id="15-4"
            key="4"
          />
          <div
            className="columns"
            id="15-5"
            key="5"
          />
          <div
            className="columns"
            id="15-6"
            key="6"
          />
          <div
            className="columns"
            id="15-7"
            key="7"
          />
          <div
            className="columns"
            id="15-8"
            key="8"
          />
          <div
            className="columns"
            id="15-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={16}
          key="16"
        >
          <div
            className="columns"
            id="16-0"
            key="0"
          />
          <div
            className="columns"
            id="16-1"
            key="1"
          />
          <div
            className="columns"
            id="16-2"
            key="2"
          />
          <div
            className="columns"
            id="16-3"
            key="3"
          />
          <div
            className="columns"
            id="16-4"
            key="4"
          />
          <div
            className="columns"
            id="16-5"
            key="5"
          />
          <div
            className="columns"
            id="16-6"
            key="6"
          />
          <div
            className="columns"
            id="16-7"
            key="7"
          />
          <div
            className="columns"
            id="16-8"
            key="8"
          />
          <div
            className="columns"
            id="16-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={17}
          key="17"
        >
          <div
            className="columns"
            id="17-0"
            key="0"
          />
          <div
            className="columns"
            id="17-1"
            key="1"
          />
          <div
            className="columns"
            id="17-2"
            key="2"
          />
          <div
            className="columns"
            id="17-3"
            key="3"
          />
          <div
            className="columns"
            id="17-4"
            key="4"
          />
          <div
            className="columns"
            id="17-5"
            key="5"
          />
          <div
            className="columns"
            id="17-6"
            key="6"
          />
          <div
            className="columns"
            id="17-7"
            key="7"
          />
          <div
            className="columns"
            id="17-8"
            key="8"
          />
          <div
            className="columns"
            id="17-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={18}
          key="18"
        >
          <div
            className="columns"
            id="18-0"
            key="0"
          />
          <div
            className="columns"
            id="18-1"
            key="1"
          />
          <div
            className="columns"
            id="18-2"
            key="2"
          />
          <div
            className="columns"
            id="18-3"
            key="3"
          />
          <div
            className="columns"
            id="18-4"
            key="4"
          />
          <div
            className="columns"
            id="18-5"
            key="5"
          />
          <div
            className="columns"
            id="18-6"
            key="6"
          />
          <div
            className="columns"
            id="18-7"
            key="7"
          />
          <div
            className="columns"
            id="18-8"
            key="8"
          />
          <div
            className="columns"
            id="18-9"
            key="9"
          />
        </div>
        <div
          className="rows"
          id={19}
          key="19"
        >
          <div
            className="columns"
            id="19-0"
            key="0"
          />
          <div
            className="columns"
            id="19-1"
            key="1"
          />
          <div
            className="columns"
            id="19-2"
            key="2"
          />
          <div
            className="columns"
            id="19-3"
            key="3"
          />
          <div
            className="columns"
            id="19-4"
            key="4"
          />
          <div
            className="columns"
            id="19-5"
            key="5"
          />
          <div
            className="columns"
            id="19-6"
            key="6"
          />
          <div
            className="columns"
            id="19-7"
            key="7"
          />
          <div
            className="columns"
            id="19-8"
            key="8"
          />
          <div
            className="columns"
            id="19-9"
            key="9"
          />
        </div>
      </div>
    </DisplayOpponentBorder>
  </Connect(DisplayOpponentBorder)>
  ,
</Provider>
`;
